cmd/conf/homi/homi.yaml

```diff
@@ -10,82 +10,93 @@ output: "homi-output"
 ######################################################################
 
-# Only for docker gen-type
-docker-image-id: klaytn/klaytn:latest
-fasthttp: true
-network-id: 2018
-no-grafana: false
-txgen: false
-txgen-rate: 2000
-txgen-conn: 100
-txgen-dur: "1m"
-txgen-th: 2
-
-rpc-port: 8551
-ws-port: 8552
-p2p-port: 32323
-data-dir: /var/klay/data
-log-dir: /var/klay/log
+deploy: 
+  rpc-port: 8551
+  ws-port: 8552
+  p2p-port: 32323
+  data-dir: /var/klay/data
+  log-dir: /var/klay/log
+  # Only for docker gen-type
+  docker: 
+    image-id: klaytn/klaytn:latest
+    fasthttp: true
+    network-id: 2018
+    no-grafana: false
+    tx-gen: 
+      enable: false
+      rate: 2000
+      connections: 100
+      duration: "1m"
+      thread: 2
 
 ######################################################################
 #                       For genesis.json                             #
 ######################################################################
+genesis: 
+  # cypress-test, cypress, baobab-test, baobab, servicechain, servicechain-test, clique, default
+  type: "default"
+  chain-id: 1000
+  service-chain-id: 1001 
+  funding-addr: ""
+ 
+  # This flags are only activated by default genesis-type
+  governance: false
+  governance-mode: "none"
+  governing-node: "0x0000000000000000000000000000000000000000"
+  unit-price: 0
+  derive-sha-implement: 0
+  reward: 
+    mint-amount: "9600000000000000000"
+    ratio: "100/0/0"
+    gini-coefficient: false
+    deferred-tx: false
+    staking-interval: 86400
+    proposer-interval: 3600
+    minimum-stake: "2000000"
+  # dynamic fee
+  kip71: 
+    lower-bound-base-fee: 25000000000
+    upper-bound-base-fee: 750000000000
+    gas-target: 30000000
+    block-gas-limit: 60000000
+    base-fee-denominator: 20
 
-# cypress-test, cypress, baobab-test, baobab, servicechain, servicechain-test, clique, default
-genesis-type: default
-chainID: 1000
-serviceChainID: 1001 
-funding-addr: ""
-
-# governance, reward and dynamic fee flags, 
-# This flags are only activated by default genesis-type
-governance: false
-gov-mode: "none"
-governing-node: "0x0000000000000000000000000000000000000000"
-unitPrice: 0
-deriveShaImpl: 0
-reward-mint-amount: "9600000000000000000"
-reward-ratio: "100/0/0"
-reward-gini-coeff: false
-reward-deferred-tx: false
-reward-staking-interval: 86400
-reward-proposer-interval: 3600
-reward-minimum-stake: "2000000"
-lower-bound-base-fee: 25000000000
-upper-bound-base-fee: 750000000000
-gas-target: 30000000
-block-gas-limit: 60000000
-base-fee-denominator: 20
-
-# Istanbul consensus engine options (default engine)
-ist-epoch: 604800
-ist-proposer-policy: 0
-ist-subgroup: 21
+  consensus: 
+    # Istanbul consensus engine options (default engine)
+    istanbul: 
+      epoch: 604800
+      proposer-policy: 0
+      subgroup: 21
 
-# To enable clique consensus engine
-clique-epoch: 604800
-clique-period: 1
+    # To enable clique consensus engine
+    clique: 
+      enable: false
```

Comment:
```go
cliqueFlag = cli.BoolFlag{
	Name:    "clique",
	Usage:   "Use Clique consensus",
	Aliases: []string{},
}
```
I made it as clique.enable but I missed the alias setting in that flag.
Thanks

Reply:
Anyway I think it's better to denote **only what we use** in the yaml file.
These files is not enough yet. We need to refine this configuration files together so that we can deploy officially.